.gitignore

```diff
@@ -0,0 +1,134 @@
+# Logs
+logs
+*.log
+npm-debug.log*
+yarn-debug.log*
```

Comment:
.gitignore file should be stored in the main branch

Reply:
Fixed